docs/visualization/metrics.md

```diff
@@ -79,6 +79,40 @@ A multi-value metrics aggregation that computes stats over numeric values extrac
 
 [Extended stats aggregation (Elasticsearch)](https://www.elastic.co/guide/en/elasticsearch/reference/current/search-aggregations-metrics-extendedstats-aggregation.html)
 
+This aggregation is mapped to :
+- [variancep()](https://docs.microsoft.com/en-us/azure/data-explorer/kusto/query/variancep-aggfunction)
+- [variance()](https://docs.microsoft.com/en-us/azure/data-explorer/kusto/query/variance-aggfunction)
+- [stdevp()](https://docs.microsoft.com/en-us/azure/data-explorer/kusto/query/stdevp-aggfunction)
+- [stdev()](https://docs.microsoft.com/en-us/azure/data-explorer/kusto/query/stdev-aggfunction)
+
+The Standard Deviation Bounds values are calculated internally by K2Bridge:
+
+```
+Lower Deviation (Population or Sampling) = Average Value - (Standard Deviation (Population or Sampling) * Sigma)
+Upper Deviation (Population or Sampling) = Average Value + (Standard Deviation (Population or Sampling) * Sigma)
+```
+
+The `Sigma` value controls controls how many standard deviations +/- from the mean should be displayed.
+
+> As reference, you can check the Elasticsearch implementation used to calculate bounds here : [getStdDeviationBound()](https://github.com/elastic/elasticsearch/blob/0699c9351f1439e246d408fd6538deafde4087b6/server/src/main/java/org/elasticsearch/search/aggregations/metrics/InternalExtendedStats.java#L187-L194)
+
+Example of Kusto Query Language built by K2Bridge:
+
+```
+let _data = kibana_data_flights
+| where (['timestamp'] >= todatetime("2007-01-27T15:18:02.1310000Z") and ['timestamp'] <= todatetime("2022-01-27T15:18:02.1310000Z"));
+
+let _extdata = _data
+| extend ['f4674a57-bb73-4916-8a3f-c0bb65ee4c0d'] = true;
+
+let _summarizablemetrics = _extdata
+| summarize ['1%extended_stats%2']=pack('count', count(),'min', min(['AvgTicketPrice']),'max', max(['AvgTicketPrice']),'avg', avg(['AvgTicketPrice']),'sum', sum(['AvgTicketPrice']),'sum_of_squares', sum(pow(['AvgTicketPrice'], 2)),'variance_population', variancep(['AvgTicketPrice']),'variance_sampling', variance(['AvgTicketPrice']),'std_deviation_population', stdevp(['AvgTicketPrice']),'std_deviation_sampling', stdev(['AvgTicketPrice'])),count() by ['f4674a57-bb73-4916-8a3f-c0bb65ee4c0d'];
+
+(_summarizablemetrics
+| project-away ['f4674a57-bb73-4916-8a3f-c0bb65ee4c0d']
+| as aggs);
+```
+
```

Comment:
At first your describe the role of the 'sigma' value, but in the adx query you can't see where the  std deviation is tied to it. I would add a paragraph on the fact this value is part of the column name + that it is later calculated (after the adx query is executed)

Reply:
Hey @eladiw 
I've made an update on the doc, and fix a small issue on double parsing !
Thank for the comment !